workflows/crisprseq_screening.nf

```diff
@@ -294,6 +296,19 @@ workflow CRISPRSEQ_SCREENING {
         }
     }
 
+    // Launch module drugZ
+    if(params.drugz) {
+        Channel.fromPath(params.drugz)
+                .splitCsv(header:true, sep:';' )
```

Comment:
Wait, you said comma separated, now you look for a semi-colon?

Reply:
I updated the documentation to make it more clear